tbx/static_src/sass/config/_variables.scss

```diff
@@ -38,6 +38,11 @@ $color--mid-grey: #282e30;
     --color--accent-five: #{$color--white};
     --color--decoration: #{$color--grey-50};
     --color--border: #{$color--grey-60};
+    // Motif heading flames
+    // - The 'fill' colour is inlined with the '#' of the hex being replaced with '%23' e.g. #ff0000 becomes %23ff0000
+    // - The hex colours used should be the same as --color--accent-two and --color--accent-three
+    --letter-motif-one: url('data:image/svg+xml;utf8,<svg xmlns="http://www.w3.org/2000/svg" width="71" height="81" viewBox="0 0 71 81"><path d="M66.805 52.031c-.374.779-1.505.626-1.67-.22-.945-5.06-3.055-9.834-5.006-11.73l-.154.457C49.657 69.905 23.035 78.79 13.979 80.956c-.861.22-1.465-.83-.855-1.473 8.336-8.564 9.757-18.821 4.668-30.974-.988-2.251-1.957-5.061-2.673-7.786 0 0-.9-3.182-1.376-7.295l-.135.067c-5.13 4.045-7.97 9.986-9.265 13.338-.324.795-1.438.761-1.688 0C1.52 43.177-.11 35.966 1.535 29.23 5.546 12.947 20.346-.12 37.883.947c.827.051 1.11 1.05.434 1.558-2.168 1.76-4.085 4.62-5.45 8.682-3.157 9.394 2.166 27.048 2.166 27.048S49.943 27.352 53.1 17.96c1.382-4.113 1.58-7.55.914-10.258-.183-.795.647-1.421 1.287-.981 14.266 9.293 16.738 26.219 14.865 35.477-.808 4.08-2.323 7.583-3.36 9.834z" fill="%237045ff" style="opacity: 0.9"/></svg>');
```

Comment:
I don't understand the `%23` in the fill colour - what does this do?

Reply:
It's replacing the `#` you'd usually use when referencing a hex. With it the `#` interferes with the correct parsing of the data and you don't see the colours. Encoding it as `%23` ensures that it is treated as a literal character in the SVG data.